packages/frontend/src/main/components/viewer/CommentAddOverlay.vue

```diff
@@ -149,6 +149,8 @@
   </div>
 </template>
 <script>
+// TODO: Need to fix the viewer package build process to be able to properly reference THREE.js
+/* global THREE */
```

Comment:
Read this for more context: https://discord.com/channels/726756379083145269/743849923803349022/959432401275215872